src/a-runtime-console/kubernetes/ui/pipeline/list/list.pipeline.component.html

```diff
@@ -23,20 +23,20 @@ <h3>
               </button>
               <ul class='dropdown-menu-right dropdown-menu' aria-labelledby='dropdownKebabRight9' role="menu" *dropdownMenu>
                 <li>
-                  <a (click)="startBuild(pipeline)" title="Start this Pipeline">Start Pipeline</a>
+                  <a class="pointer" (click)="startBuild(pipeline)" title="Start this Pipeline">Start Pipeline</a>
```

Comment:
In the Codebases page this `<a>` tag has `href="javascript:void(0)" class="dropdown-item secondary-action"` in it. It might be worth doing the same thing here instead. Or we should update that one too.

Reply:
Semantically a link with href should be used only for navigating the app.

A button should be used for executing actions.

Buttons also do not show a pointer cursor by default. They also differ from links in that they can be clicked with the space button.

Overloading the href to execute js or assigning href value  to # is something I'd avoud. I suggest keeping the CSS to assign a pointer cursor.

Reply:
That is where my thoughts started to head too, so we should update the other one too. The one in Codebases. 

We also need to look at the dropdown component, like you said.